chat-plugins/calculator.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+function isNumeric(str) {
+	return !isNaN(parseFloat(str));
+}
+
+function parseMathematicalExpression(infix) {
+	// Shunting-yard Algorithm -- https://en.wikipedia.org/wiki/Shunting-yard_algorithm
+	let outputQueue = [];
+	let operatorStack = [];
+	let operators = {
```

Comment:
I would pull this out into the parent scope, and make it a constant. Maybe even put it in all-uppercase, like `const OPERATORS`.

Reply:
> I would pull this out into the parent scope.

What do you mean by this? It's already in the parent scope, if you meant _outside_ of the function (global scope), it's not needed because only ``parseMathematicalExpression`` uses it.

Reply:
done